tests/test_point_range.py

```diff
@@ -0,0 +1,89 @@
+"""
+Tests for Point.
+
+To run this unit test install UnitTesting package and choose
+"UnitTesting: Test current project" command.
+"""
```

Comment:
I understand we may want to test _in_ sublime using the UnitTesting package eventually, but these tests should work with the standard python tools.

Reply:
I never leave ST for developement, so this is the best way I found.